<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRecognize</class>
 <widget class="QWidget" name="FormRecognize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_ReadImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>读取图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GrabImage">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>相机采图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TestImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>测试图片</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>251</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_CheckNum">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>250</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DrawPos1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制定位1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DrawPos2">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>100</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制定位2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DrawDetect">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开样图文件夹</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>检测筷子根数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveSample">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存样品图</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>检测搜索框宽</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CheckWidth">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CheckHeight">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>检测搜索框高</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>建议高是宽</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>的8倍</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Train">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>训练</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_SaveImage">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
